doc/man1/openssl-req.pod

```diff
@@ -342,13 +342,13 @@ for key generation operations.
 
 =item B<-sm2-id>
 
-Specify the ID string to use when verifying an SM2 certificate. The ID string is
-required by the SM2 signature algorithm for signing and verification.
+Specify the ID string to use when verifying an SM2 certificate request. The ID
+string is required by the SM2 signature algorithm for signing and verification.
 
 =item B<-sm2-hex-id>
 
-Specify a binary ID string to use when signing or verifying using an SM2
-certificate. The argument for this option is string of hexadecimal digits.
+Specify a binary ID string to use when verifying an SM2 certificate request.
+The argument for this option is string of hexadecimal digits.
```

Comment:
This change seems quite unrelated. Probably should have been a different PR - but at least in a different commit?

Reply:
No problem, will do that.

Reply:
Fixed, move the doc change to another commit